### DISCUSSION TITLE
imported leveling guide for campaign, does not show anything on screen

### DISCUSSION THREAD

**Discussion by callmexile** [Lailloken/Exile-UI#330] [General]
Unsure what im doing incorrectly in order to get my leveling tracker to work including the zone overlay. I have the guide generated and nothing working. Any help would be appreciated!

**Comment by Lailloken** [comment]
Generally, you export from exile-leveling, import it in the `leveling tracker` section of the settings menu, and then click the button with the book icon to toggle the guide-overlay on/off.
There should be error messages on both the `import` button and the book button if there's an issue.

**Reply by callmexile** [reply]
i am not understanding how to export the data onto the level tracking menu. i downloaded the zip file from exile leveling, canont find a export.  then if i try click import on level tracker is says none is found.

**Reply by Lailloken** [reply]
If everything in exile-leveling is set up, you have to open the menu and click this:
![image](https://user-images.githubusercontent.com/61888437/214361526-41ea39db-9ec9-40c1-90ce-f0a366975b37.png)

Then go back to my tool and click import
